src/SinglyLinkedList.java

```diff
@@ -0,0 +1,20 @@
+public class SinglyLinkedList {
+    public Node n;
```

Comment:
You are using here a Node from the Java library. This is not what you want to use